translations/translations.json

```
@@ -6718,6 +6718,7 @@
       "onlyCurrentlyOpen": "nur aktuell geöffnete Orte anzeigen",
       "poiCategories": "Kategorien",
       "showPois": "{{numberOfPois}} Orte anzeigen",
+      "showPois_one": "{{numberOfPois}} Ort anzeigen",
```

You should use something like this:
```suggestion
      "showPois_zero": "0 Orte gefunden",
      "showPois_one": "1 Ort anzeigen",
      "showPois_two": "{{count}} Orte anzeigen",
      "showPois_few": "{{count}} Orte anzeigen",
      "showPois_many": "{{count}} Orte anzeigen",
      "showPois_other": "{{count}} Orte anzeigen",
```

Then you can just use
``` ts
t('showPois', { count: poisCount })
```
Handling for zero pois is already handled then so you can furthermore simplify the rendering probably :)

---

Alright, I’ll use that instead.